_includes/configuration/formats.md

```diff
@@ -319,6 +319,40 @@ tinymce.init({
 });
 ```
 
+#### `merge_siblings`
+
+When the `merge_siblings` parameter is set to `false`, two or more similar inline elements will not automatically merge.
+
+When `merge_siblings` is set to `true` the following two HTML elements will be merged automatically.
+
+Before merge:
+
+```html
+<span class="myclass">some</strong><span class="myclass">text</strong>
```

Comment:
Is this meant to be incorrect HTML? I don't really know enough about this setting to know what's going on

Reply:
:facepalm: no I changed the opening tags and forgot to update the closing tags

Reply:
Fixed by https://github.com/tinymce/tinymce-docs/pull/1832/commits/e5d3a632db43d0a93436be0dbfeefd7d33e0d607